RELEASING.md

```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
 - [ ] Tag and push merged release commit.
   - CI will trigger and create a draft release with assets attached.
 - [ ] Prepare extension release notes, share with team asking for feedback.
-- [ ] Manually tag and push `timescaledev/promscale-extension` docker image (see #354)
 - [ ] Attach release notes to draft release created above.
-- [ ] Publish the GitHub release
+- [ ] Publish the GitHub release.
 
 ## Post-Release
 - [ ] Prepare for the next development cycle:
   - Generate SQL schema and commit it (and upgrade symlinks to it) to the `sql` directory.
+<details>
+<summary>Examples</summary>
+
+Generate SQL file:
+
+```bash
+cargo pgx schema --release
+mv sql/promscale-0.5.5.sql sql/promscale--0.5.5.sql
+```
+
+Create symlinks:
+
+```bash
+for i in {0..4}; do ln -s sql/promscale--0.5.5.sql sql/promscale--0.5.${i}--0.5.5.sql; done
+```
+
+Force add the `.sql` files to the commit, as they're in `.gitignore`:
+
+```bash
+git add sql/*--0.5.5*.sql --force
```

Comment:
```suggestion
git add --force sql
```

Reply:
I'm afraid this will cause unwanted files to be added. Like the `promscale-0.x.x.sql` (the single hyphen) file.

Reply:
Ya, you're right. As discussed, you can use `extract-extension-version.sh`.